prime-router/settings/prod/0140-NJ-Allow-Lab-name-for-MSH-4.yml

```diff
@@ -0,0 +1,317 @@
+---
+- name: "nj-doh"
+  description: "New Jersey Department of Health"
+  jurisdiction: "STATE"
+  stateCode: "NJ"
+  countyName: null
+  filters:
```

Comment:
We should think about whether these should be under the receivers instead of the org as not all the filters appear to be working as intended